src/components/view/ActionButton.vue

```diff
@@ -44,7 +44,7 @@
         v-if="action.api in $store.getters.apis &&
           !action.showBadge &&
           ((!dataView && (action.listView || action.groupAction && selectedRowKeys.length > 0)) || (dataView && action.dataView)) &&
-          ('show' in action ? action.show(resource, $store.getters.userInfo) : true)"
+          ('show' in action ? action.show(resource, $store.getters.userInfo, $store.getters.isLdapEnabled) : true)"
```

Comment:
I'm thinking to get rid of this at this.$store must be accessible by all parts of the code; or simply only pass $store.